cross/sslh/Makefile

```diff
@@ -1,42 +1,26 @@
 PKG_NAME = sslh
-PKG_VERS = 1.13
-SRC_VERSION = 1.13b
+PKG_VERS = v1.16
+SRC_VERSION = v1.16
```

Comment:
Seems this is no longer needed, right? Otherwise, using `${PKG_VERS}` would be just as good.